src/routes/ConfigureGame/index.tsx

```diff
@@ -204,6 +212,41 @@ export default function ConfigureGame({ quizId, userName = 'guest' }) {
               setValue('isQuestionPointsHidden', !isQuestionPointsHidden);
             }}
           />
+          <Group position="apart" mb="xl">
+            <Checkbox
+              radius="xl"
+              size="md"
+              mb="xs"
+              ml="md"
+              label="Allow negative response for incorrect response"
+              checked={negativePointsForIncorrect !== 0}
+              onChange={() => {
+                if (negativePointsForIncorrect === 0) {
+                  setValue('negativePointsForIncorrect', -1);
```

Comment:
We can even restrict it to some fixed values if we do a dropdown instead "50%", "25%", "75%" etc.

Reply:
Also after calculating score make sure to round the score to 2 decimals.

Reply:
percentage will make the logic more complicate and it will not be user friendly. Later we can allow user do upload a spreadsheet which has different negative points for different questions and it will be a paid feature.
We shouldn't provide everything for free to users

Reply:
I was thinking to make `negative points` feature as paid, as it is not a required feature and users who are using our app for professional purpose will use this feature

Reply:
Yeah, we can make it paid feature later. Initially every feature will be free until we get enough users. We can roll out a subscription model later.

But, percentage will not be confusing to users. It is usually happens that way only. Uploading csv for different negative points will be much more painful rather than just adding a percentage that works for every question automatically.

Reply:
For simplicity we can just let them choose some default options only: "1/2", "1/4", "1/3". So, instead of input field we show a select with these 3 options.

Reply:
Below we show a note: "If question point is 100 then upon answering incorrectly, we deduct 50 points" when they select 1/2 for example.

Reply:
So, it will be more clear